samza-core/src/main/scala/org/apache/samza/util/Util.scala

```diff
@@ -49,7 +49,10 @@ object Util extends Logging {
 
   /**
    * Instantiate an object of type T from a given className.
+   *
+   * Deprecated: Use [[ReflectionUtil.getObj(String, Class, ClassLoader)]] instead.
```

Comment:
Consumers of this method are not aware of which class loader to use. Can you add a recommendation here as part of deprecated notes?